src/tools/rest-frontend/resources/structure.json.in

```diff
@@ -0,0 +1,240 @@
+[
+    {
+        "name": "Documentation",
+        "type": "submenu",
+        "ref": "documentation",
+        "children": [
+            {
+                "name": "Getting started",
+                "type": "staticlist",
+                "ref": "docgettingstarted",
+                "children": [
+                    {
+                        "name": "Why Elektra?",
+                        "type": "staticfile",
+                        "options": {
+                            "path": "doc/WHY.md"
+                        }
+                    },
+                    {
+                        "name": "Big Picture",
+                        "type": "staticfile",
+                        "options": {
+                            "path": "doc/BIGPICTURE.md"
+                        }
+                    },
+                    {
+                        "name": "Compiling",
+                        "type": "staticfile",
+                        "options": {
+                            "path": "doc/COMPILE.md"
+                        }
+                    },
+                    {
+                        "name": "Installation",
+                        "type": "staticfile",
+                        "options": {
+                            "path": "doc/INSTALL.md"
+                        }
+                    },
+                    {
+                        "name": "Tutorials",
+                        "type": "staticref",
+                        "options": {
+                            "path": "tutorials"
+                        }
+                    }
+                ]
+            },
+            {
+                "name": "Tutorials",
+                "type": "parsereadme",
+                "ref": "tutorials",
+                "options": {
+                    "path": "doc/tutorials/README.md",
+                    "target_file": [],
+                    "parsing": {
+                        "stop_regex": "####### UNUSED ######",
+                        "section_regex": "## ([^#]+)",
+                        "entry_regex": "^\\- \\[(.+)\\]\\(([^\\)]+)\\)(.*)"
+                    },
+                    "name": {
+                        "make_pretty": false
+                    }
+                }
+            },
+            {
+                "name": "Plugins",
+                "type": "parsereadme",
+                "ref": "plugins",
+                "options": {
+                    "path": "src/plugins/README.md",
+                    "target_file": ["README.md", "README", "readme.md", "readme"],
+                    "parsing": {
+                        "start_regex": "# Plugins #",
+                        "stop_regex": "####### UNUSED ######",
+                        "section_regex": "### ([^#]+) ###",
+                        "entry_regex": "^\\- \\[(.+)\\]\\(([^\\)]+)\\)(.*)"
+                    },
+                    "name": {
+                        "make_pretty": false
```

Comment:
Thanks for the explanation. Is it possible to have false as default and leave it away here? In any case, please add the info to "info".